src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -2430,7 +2564,13 @@ window.qBittorrent.DynamicTable ??= (() => {
                 row.full_data.remaining = (row.full_data.size * (1.0 - (row.full_data.progress / 100)));
         },
 
-        setupCommonEvents: () => {}
+        setupCommonEvents: function() {
+            if (this.useVirtualList) {
+                this.dynamicTableDiv.addEventListener("scroll", () => {
```

Comment:
Always add use an event variable even if you don't need it:
(Please apply it globally to your new code)
```suggestion
                this.dynamicTableDiv.addEventListener("scroll", (e) => {
```